rubin_sim/maf/metrics/sn_n_sn_metric.py

```diff
@@ -543,9 +542,9 @@ def duration_z(self, grp, min_duration=60.0):
 
         Parameters
         --------------
-        grp: pandas df group
+        grp : pandas df group
```

Comment:
This is a contributed metrics that is pretty complicated. TBH, I'm not sure we understand all the ins and outs either. Peter tried to rewrite them without pandas at one point and gave up because what is happening where is complicated enough that it's not super clear. 
I know that the docstrings here haven't been updated properly, but I'm not sure it's an easy task either. 
As part of my idea here to "get done what I can", I think I'm going to pass on making sure this metric docstrings properly and just make sure it gets through ruff. I would love it if the contributor would fix it properly.

Reply:
Ok, not sure if I didn't make this worse ... but it's a thing.